sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -233,28 +260,37 @@ export class DataLakeSASPermissions {
     add: boolean;
     create: boolean;
     delete: boolean;
+    execute: boolean;
+    move: boolean;
+    ownership: boolean;
     static parse(permissions: string): DataLakeSASPermissions;
+    permission: boolean;
```

Comment:
Rename. Follow other languages. Follow python.

Reply:
Didn't find them in python. Following .NET. Renamed `ownership` to `manageOwnership`, `permission` to `manageAccessControl`.